tasks/stage-product/task.sh

```diff
@@ -24,7 +24,7 @@ function main() {
 
   local version
   pushd "${cwd}/pivnet-product"
-    version="$(ls -1 *.pivotal | sed "s/"${PRODUCT_NAME}"-\(.*\).pivotal/\1/")"
+    version="$(unzip -p *.pivotal 'metadata/*.yml' | cat | grep 'product_version:' | cut -d ':' -f 2 | tr -d ' ')"
```

Comment:
Kris, please feel free to reject this PR. I probably picked the wrong commit out of the forked repo when creating the PR. I will resubmit another one. Sorry about that.